src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,112 @@
+module Algebra.Graph.Acyclic.AdjacencyMap where
+
+import qualified Algebra.Graph.AdjacencyMap as AM
+import qualified Algebra.Graph.AdjacencyMap.Algorithm as AMA
+import qualified Algebra.Graph.AdjacencyMap.Internal as AMI
```

Comment:
For simplicity, I think we should use `AM` for qualifying all these modules, because they all refer to the same `AdjacencyMap` data type and have no name clashes.

Reply:
Ok, will change that!